src/rpc/server.cpp

```diff
@@ -425,6 +426,21 @@ static inline JSONRPCRequest transformNamedArguments(const JSONRPCRequest& in, c
         } else {
             hole += 1;
         }
+        if (out.params.size() == 0) skipped += 1;
+    }
+    // If leftover "args" param was found, use it as a source of positional
+    // arguments and add named arguments after.
```

Comment:
Outside of this PR changeset, it may not be clear where "args" comes from. It may be good to mention in the comment here that "args" is set in `rpc/client.cpp::RPCConvertNamedValues()`.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/19762#discussion_r525090389

> Outside of this PR changeset, it may not be clear where "args" comes from. It may be good to mention in the comment here that "args" is set in `rpc/client.cpp::RPCConvertNamedValues()`.

Thanks, if I follow up I'll reference [JSON-RPC-interface.md#parameter-passing](https://github.com/ryanofsky/bitcoin/blob/pr/named/doc/JSON-RPC-interface.md#parameter-passing) here. Better to reference the documentation than the implementation details of one specific client, though. Referencing the `RPCConvertNamedValues` function where "args" is set in the CLI client would be equivalent to referencing `AuthServiceProxy.get_request` method where "args" is set in the python client. These would be potentially helpful comments, but also likely to become out of date.